spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -29,6 +29,9 @@ describe('ApplicationNewPage', () => {
 
     // Save the application
     await page.click('.save-btn')
+    // Verify that no errors are present on save
```

Comment:
Everything looks good but this block of code is in here a number of times so maybe we should make a simple helper function even just at the top of this file so we do not have the duplicate blocks throughout